app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -101,13 +101,39 @@ def csv_file_download_path(run)
       )
     end
 
+    # Returns the appropriate value for a Proc, lambda, symbol, or other types.
+    #
+    # @param validator [ActiveModel::Validations::*] The validator object used to resolve the value.
+    # @param record [Task] The record for which the value needs to be resolved.
```

Comment:
That makes sense -- applied in https://github.com/Shopify/maintenance_tasks/pull/962/commits/4eec6124987b914fd8a8c3c87e6703475944b001
(I used record because I had been looking [at this](https://github.com/rails/rails/blob/6b2a9f1213d7aa4f9a25db7640ad68ec0d8c31ce/activemodel/lib/active_model/validations/resolve_value.rb#L6))